unittests/test-contracts/test_api/test_checktime.cpp

```diff
@@ -7,6 +7,8 @@
 #include "test_api.hpp"
 using namespace sysio;
 
+using namespace sysio;
```

Comment:
I don't see why we would need this in the file, since there are no other changes. Remove

Reply:
yeah, it's a duplication w/ line 8, just a bad merge artifact.